### PR TITLE
Homey Extension

### PR DESCRIPTION
**Pull request by DominicVonk**
## Description

Added homey extension

## Screencast

https://user-images.githubusercontent.com/3958086/195829062-c259fc4c-d435-4d14-ae90-1edf9e29f044.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/homey/src/flows.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                      source: flow.triggerable && flow.enabled ? Icon.PlayFilled : Icon.XMarkCircleFilled,
+                      tintColor: flow.triggerable && flow.enabled ? Color.Green : Color.Red,
+                    }}
+                    title={flow.name}
+                    actions={
+                      <ActionPanel title={flow.name}>
+                        <ActionPanel.Section>
+                          {flow.triggerable && flow.enabled && (
+                            <Action
+                              title="Start Flow"
```
Comment (pernielsentikaer) [inline extensions/homey/src/flows.tsx:42]: Missing an icon here

File: extensions/homey/src/devices.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                          ]
+                        : []
+                    }
+                    actions={
+                      <ActionPanel title={device.name}>
+                        <ActionPanel.Section>
+                          {device?.capabilitiesObj?.onoff && (
+                            <>
+                              <Action
+                                title="Toggle"
```
Comment (pernielsentikaer) [inline extensions/homey/src/devices.tsx:74]: Missing an icon here

File: extensions/homey/src/devices.tsx [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+                            </>
+                          )}
+                          <Action.OpenInBrowser
+                            title="Goto Device"
+                            url={"https://my.homey.app/homeys/" + homey.getHomey().id + "/devices/" + device.id}
+                          ></Action.OpenInBrowser>
+                          {device?.capabilitiesObj?.onoff && (
+                            <>
+                              <Action
+                                title="On"
```
Comment (pernielsentikaer) [inline extensions/homey/src/devices.tsx:94]: Missing an icon here

File: extensions/homey/src/devices.tsx [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+                                  await showToast({
+                                    title: "Device turned on",
+                                    message: device.name,
+                                    style: Toast.Style.Success,
+                                  });
+                                  setIndex(index + 1);
+                                }}
+                              ></Action>
+                              <Action
+                                title="Off"
```
Comment (pernielsentikaer) [inline extensions/homey/src/devices.tsx:106]: Missing an icon here

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension and I have some feedback ready for you:

* You should use the [isLoading property](https://developers.raycast.com/api-reference/user-interface/list#props) on the lists 

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by DominicVonk** [comment]
I added the icons and the loading

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**